docs/build/basics/Node Economics.md

```diff
@@ -0,0 +1,20 @@
+# Node Operator Economics: An Illustration
+
```

Comment:
Can you add some commentary here? e.g "Node operators help secure the network. Here is a quick explanation using a simple example of what to expect in terms of the node economics."